consumatio/usecases/episode_details.py

```diff
@@ -12,7 +12,7 @@ def episode_details(tmdb, code, season_number, episode_number):
         "overview": dict_episode_details.get("overview"),
         "airDate": dict_episode_details.get("air_date"),
         "voteAverage": dict_episode_details.get("vote_average"),
-        "stills": dict_episode_images.get("stills"),
+        "still": dict_episode_images.get("stills"),
```

Comment:
This applies to those below as well, are you sure it is `dict_episode_images.get("stills")` and not `dict_episode_images.get("still")`?

Reply:
Yes! The internal representation was an array which got reduced to a single still. Thats why there is a difference in the naming.

Reply:
Ah, thanks for the clarification!